__docs__/wonder-blocks-clickable/clickable.stories.tsx

```diff
@@ -337,13 +339,13 @@ const styles = StyleSheet.create({
         marginRight: spacing.large_24,
     },
     navigation: {
-        border: `1px dashed ${color.purple}`,
+        border: `1px dashed ${semanticColor.border.primary}`,
         marginTop: spacing.large_24,
         padding: spacing.large_24,
     },
     disabled: {
-        color: color.white,
-        backgroundColor: color.offBlack64,
+        color: semanticColor.text.inverse,
+        backgroundColor: semanticColor.surface.overlay,
```

Comment:
Should we use `semanticColor.text.disabled`? Would it be helpful to have a different token for the background color since the clickable isn't an overlay?

Reply:
I didn't want to use `text.disabled` because it's being used in the background. I think I'm going to change this to use `action.disabled` instead.